### PR TITLE
minor details in graph_plot

### PR DESCRIPTION
**Pull request by fchapoton**
just fixing a few little things in the modified file

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/graphs/graph_plot.py [hunk 1]
```diff
@@ -168,8 +168,7 @@
         'Which graphviz layout program to use -- one of '
         '"circo", "dot", "fdp", "neato", or "twopi".',
     'by_component':
-        'Whether to do the spring layout by connected component -- a boolean.',
-    }
+        'Whether to do the spring layout by connected component -- a boolean.'}
```
Comment (kwankyu) [inline src/sage/graphs/graph_plot.py:172]: I am not sure if this is an improvement. Is there a standard regarding this?

**Comment by kwankyu** [comment]
Otherwise, lgtm.

File: src/sage/graphs/graph_plot.py (hunk 1 above)
Comment (fchapoton) [reply src/sage/graphs/graph_plot.py:172]: this is fixing a pep8 warning issued by pycodestyle, but not one already activated in our linter

Comment (kwankyu) [reply src/sage/graphs/graph_plot.py:172]: OK, then. Thanks.

**Review by kwankyu — APPROVED** [review]

**Pull request merged**